plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -553,6 +554,17 @@ def download_path(self):
         """
         return self.filename
 
+    def verify_size(self, location):
```

Comment:
Both this class and the other class you added the function to inherit from the same class so this method should probably live on the parent class.